client/js/admin.js

```diff
@@ -27,11 +27,11 @@ const submit = function() {
 
 	$("#status").text("Uploading...")
 
-	const XHR = new XMLHttpRequest()
-	XHR.open("POST", "")
-	XHR.setRequestHeader("Content-Type", "application/json")
+	const xhr = new XMLHttpRequest()
```

Comment:
The general convention I've seen (outside the JS API) is that acronyms are single words, and thus get camel-cased as such.  So you would have `xhr` and `createNewXhr`, for example.  We could go a different way if you feel strongly.